lib/friendly-errors/transformers/missing-loader.js

```diff
@@ -32,7 +32,7 @@ function isErrorFromVueLoader(filename) {
     }
 
     // vue3
-    if (filename.includes('vue-loader/dist??')) {
+    if (/vue-loader\/dist(\/index\.js)?\?\?/.test(filename)) {
```

Comment:
Looks like the filename now includes `/index.js`.
I don't know if that's always the case so I also kept supporting the version without it.